docs/user_manual.md

```diff
@@ -332,6 +325,43 @@ These settings correspond to the ones in the [Docker docs][2].
 
 What is allowed here will vary based upon the discovery service plugin used.
 
+#### rolloutOptions
+When you run `helios rolling-update`, the rolling-update options will be constructed from the
+values you provide on the CLI and the job configuration with the following precedence:
+
+* Parameters included in the API request (eg, flags passed on the command line)
+* Parameters defined in the job configuration (of the job being rolled out)
```

Comment:
how about adding something like "see above for an example of what this looks like in JSON"?

Reply:
ah forgot that part.